tasks/symlink-shared.yml

```diff
@@ -6,10 +6,19 @@
     path: "{{ ansistrano_release_path.stdout }}/{{ item }}"
   with_items: "{{ ansistrano_shared_paths }}"
 
+# Ensure symlink shared paths are existing
+- name: ANSISTRANO | Ensure shared paths directories are existing
```

Comment:
My second choice of words was: 'are created' if you don't mind the current naming.